CardForm/src/test/java/com/braintreepayments/cardform/view/CardFormTest.java

```diff
@@ -1918,7 +1919,7 @@ public void valuesAreRestored() {
         assertEquals("1 234-567-8", ((EditText) mActivity.findViewById(R.id.bt_card_form_mobile_number)).getText().toString());
     }
 
-    @Test
+    @Ignore("card.io is no longer maintained")
```

Comment:
These are doing nothing, why not just delete them?

Reply:
#80

Reply:
We were planning to remove them at the same time we remove CardIO entirely, to keep the PR history more consistent.

Reply:
In that case this should likely be a separate commit outside this PR then